specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -920,6 +1549,155 @@
           }
         }
       }
+    },
+    "AcsRouterChannelConfiguration": {
+      "description": "Router Channel Configuration",
+      "type": "object",
+      "properties": {
+        "channelId": {
+          "description": "Channel ID for Router Job",
+          "type": "string"
+        },
+        "capacityCostPerJob": {
+          "description": "Capacity Cost Per Job for Router Job",
+          "type": "integer",
+          "format": "int32"
+        },
+        "maxNumberOfJobs": {
+          "description": "Max Number of Jobs for Router Job",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "AcsRouterQueueDetails": {
+      "description": "Router Queue Details",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Router Queue Id",
+          "type": "string"
+        },
+        "name": {
+          "description": "Router Queue Name",
+          "type": "string"
+        },
+        "labels": {
+          "description": "Router Queue Labels",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AcsRouterCommunicationError": {
+      "description": "Router Communication Error",
+      "properties": {
+        "code": {
+          "description": "Router Communication Error Code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Router Communication Error Message",
+          "type": "string"
+        },
+        "target": {
+          "description": "Router Communication Error Target",
+          "type": "string"
+        },
+        "innerError": {
+          "description": "Router Communication Inner Error",
+          "$ref": "#/definitions/AcsRouterCommunicationError"
+        },
+        "details": {
+          "description": "List of Router Communication Errors",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterCommunicationError"
+          }
+        }
+      }
+    },
+    "AcsRouterWorkerSelector": {
+      "description": "Router Job Worker Selector",
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "Router Job Worker Selector Key",
+          "type": "string"
+        },
+        "labelOperator": {
+          "description": "Router Job Worker Selector Label Operator",
+          "type": "string",
+          "enum": [
+            {
+              "operation": "=",
```

Comment:
Where did you see this property documented? I don't think this is valid.

Reply:
If you want to add descriptions, you would need to do it like https://github.com/Azure/azure-rest-api-specs/pull/24153/files#diff-4b49e91c6d3dc4a0bf20909734cce568db54e4721a79a675e92c1dcb6440c53eR1686